src/menu/src/MenuOption.js

```diff
@@ -25,15 +25,7 @@ const MenuOption = memo(function MenuOption(props) {
 
   const handleClick = useCallback(e => onSelect(e), [onSelect])
 
-  const handleKeyPress = useCallback(
-    e => {
-      if (e.key === 'Enter' || e.key === ' ') {
-        onSelect(e)
```

Comment:
Same comment / question re: `onSelect` missing in the `onKeydown`

Reply:
`onSelect` isn't needed. That's the whole point of `use-clickable` – you dont have to manually duplicate "click" functionality. It enables other elements (aside from ones that support it OOTB like `button`) to trigger click events for keyboard clicks (Enter/Space). This behaves the same as how native buttons work.